eWonConnector-build/pom.xml

```diff
@@ -48,7 +48,7 @@
                     <moduleName>${project.parent.name}</moduleName>
                     <moduleDescription>${project.description}</moduleDescription>
                     <moduleVersion>1.1.12</moduleVersion>
-                    <requiredIgnitionVersion>8.0.0</requiredIgnitionVersion>
+                    <requiredIgnitionVersion>8.1.0</requiredIgnitionVersion>
```

Comment:
Curious as to why the required ignition version is less than the sdk version. is  sdk v8.1.1 supported in v8.1.0?

Reply:
Not sure, but I just made the required version the same as the SDK version